src/picongpu/submit/pizdaint-cscs/SLURM_Tutorial.md

```diff
@@ -0,0 +1,27 @@
+SLURM examples
```

Comment:
the `-t` template and the hyperthreading difference are not that crucial. ht can be documented in the alias :)